BinaryEncoding.md

```diff
@@ -425,18 +425,16 @@ It is legal to have several entries with the same type.
 | Name | Opcode | Immediates | Description |
 | ---- | ---- | ---- | ---- |
 | `unreachable` | `0x00` | | trap immediately |
-| `block` | `0x01` | sig : `inline_signature_type` | begin a sequence of expressions, yielding 0 or 1 values |
```

Comment:
This doesn't match your "alternative opcode proposal". BLock has value 0x2 (nop is 0x1).